website/docs/r/cdn_frontdoor_rule_sets_association.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_rule_sets_association"
+description: |-
+  Manages the association between one or more Front Door (standard/premium) Rule Sets and a Front Door (standard/premium) Route.
+---
+
+# azurerm_cdn_frontdoor_rule_sets_association
+
+Manages the association(s) between one or more Front Door (standard/premium) Rule Sets and a Front Door (standard/premium) Route. Use this resource to avoid receiving the service side error `This resource is still associated with a route. Please delete the association with the route first before deleting this resource` when you attempt to `destroy`/`delete` a Front Door (standard/premium) Rule Set. 
```

Comment:
We can get rid of this note as it will be the only way to associate these two resources soon